### PR TITLE
22.12.1 Documentation Changes

### PR DESCRIPTION
**Pull request by Mrt134**
Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

**Review by Mrt134 — COMMENTED** [review]
First polish pass completed.

File: content/SCALE/GettingStarted/Install/InstallEnterpriseHASCALE.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+   Select **Reboot standby TrueNAS controller** and **Confirm**, then click **Proceed** to start the sync operation. This sync controller B with controller A which adds the network settings and pool to controller B.
+
+   ![FailoverSyncToPeerDialog](/images/SCALE/22.12/FailoverSyncToPeerDialog.png "Failover Sync To Peer")
+
+When the system comes back up, log into SCALE using the virtual IP address. The main **Dashboard** should now have two **System Information** widgets, one for controller A with the serial number and the host name that includes the letter **a** and the other for controller B labeled as **Standby Controller** and that includes the serial number and the host name that includes the leter **b**. Take note of this information.
+
+![HAMainDashboard](/images/SCALE/22.12/HAMainDashboard.png "Main Dashboard for HA Systems")
+{{< /expand >}}
+
+{{< taglist tag="scaleinstall" title="Related Installation Articles" >}}
```
Comment (clairewrtr) [inline content/SCALE/GettingStarted/Install/InstallEnterpriseHASCALE.md:213]: No related articles displaying?

File: content/SCALE/GettingStarted/Install/SetUpSharing.md [hunk 2]
```diff
@@ -50,7 +48,7 @@ To set up SMB sharing:
       
       ![AddUserUserIDAndGroupsSettings](/images/SCALE/22.12/AddUserUserIDAndGroupsSettings.png "Add User User Id an Groups Settings")
           
-   b. Enter the values in each required field, and then verify the checkmark for **Samba Authentication** exists.
+   b. Enter the values in each required field, and then verify **Samba Authentication** is selected.
```
Comment (clairewrtr) [inline content/SCALE/GettingStarted/Install/SetUpSharing.md:51]: Is this the correct image for "Add User" as it has "Permit Sudo" checkbox.

File: content/SCALE/GettingStarted/Install/UIConfigurationSCALE.md [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+{{< /enterprise >}}
+
+### Data Storage Backup Solutions
+After saving both files, we recommend you set up snapshots, replication, or a cloud storage provider service and cloud sync task to back up your storage data.
+{{< expand "Backup Solutions" "v" >}}
+
+TrueNAS SCALE provides options to back up your stored data:
+* Snapshots and periodic snapshot tasks
+* Replication to another system
+* Cloud storage provider options and cloud sycn tasks
```
Comment (clairewrtr) [inline content/SCALE/GettingStarted/Install/UIConfigurationSCALE.md:171]:
```suggestion
* Cloud storage provider options and cloud sync tasks 
```

File: content/SCALE/GettingStarted/Install/UIConfigurationSCALE.md [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+TrueNAS SCALE main **Dashboard** displays an image of the TrueNAS system server on the **System Information** widget. Click on the image to open the **[View Enclosure]({{< relref "EnclosureScreensSCALE" >}})** screen, or select the **System Settings > Enclosure** option on the main menu navigation panel.
+
+The **View Enclosure** screen provides details about the system disks, the pools and VDEVs they are in, the hardware, and the disk status. Click on a drive to view the disk details.
+If the TrueNAS system is equipped with an expansion shelf, an image of the expansion shelf with the disks populated in it also displays.
+
+If using SCALE on servers not provided by TrueNAS, the main **System Information** widget on the main **Dashboard** displays the TrueNAS SCALE logo and the **System Settings > Enclosure** option does not exist on the menu navigation panel.
+
+## Using Applications in SCALE
+SCALE provides a list of applications you can deploy on the **Apps > Available Applications** screen.
+SCALE Enterprise only offers MinIO as an available application.
```
Comment (clairewrtr) [inline content/SCALE/GettingStarted/Install/UIConfigurationSCALE.md:209]:
```suggestion

```

**Review by clairewrtr — APPROVED** [review]
Looks Good.

**Pull request merged**